resources/[soz]/soz-core/src/server/job/ffs/ffs.restock.provider.ts

```diff
@@ -164,7 +164,18 @@ export class FightForStyleRestockProvider {
                     }
                 }
             }
-            const randomItem = loopItems[Math.floor(Math.random() * loopItems.length)];
+
+            const itemStock = loopItems.filter(
+                item => item.stock != null && item.stock < 50
+            );
+            
+            let randomItem;
+            if (!itemStock) {
```

Comment:
```suggestion
            if (!itemStock.length) {         
```

Ca marche en python, mais pas en JS :pray:

Reply:
https://github.com/Pim-612/SOZ-FiveM-Server/blob/7604e67b36fbc5978ca833e058d734d7cd6bbc79/resources/%5Bsoz%5D/soz-core/src/server/job/ffs/ffs.restock.provider.ts#L179

Ah mince! X) je pensais que ça marchais pcq j'avais vu cette ligne en dessous

Reply:
Ah mince! X) je pensais que ça marchais pcq j'avais vu cette ligne (Je suis néophyte en TS )
" if (!randomItem || !loopItems) {" (ligne 179)